src/curl.c

```diff
@@ -725,6 +725,9 @@ CURLcode swupd_curl_set_basic_options(CURL *curl, const char *url, bool fail_on_
 		goto exit;
 	}
 
+        curl_easy_setopt(curl, CURLOPT_USERAGENT, PACKAGE "/" VERSION);
```

Comment:
Make sure you run `make compliant` on your patch to make sure you are compliant with the code style being used. Looks like you are using spaces here over tab.

Reply:
I did not realize TAB was still a thing =D. Thanks for educating me though. I'm force pushing a fix to my code style error.